src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { Parser, ParseResult } from './parser/parser';
+export {
+  ContractSchemasClient,
+  ContractSchemas,
+} from './parser/contract-schemas.client';
+export { Schemas, Schema, parseSchemasFromBytes } from './parser/schema';
+export { Event } from './parser/event';
```

Comment:
We should restructure the sources to have `casper` internally in the `parser` since it's used here and is not developed in parallel as a monorepo.

Will the build process come in a separate PR?